README.md

```diff
@@ -8,6 +8,11 @@ Please see the [MET website](https://dtcenter.org/community-code/model-evaluatio
 Version History
 ===============
 
+v3.3
```

Comment:
I would remove this because we decided to stay on v3.2 for this additional because the base image has not changed.

Reply:
You could include the date when you added the metviewer image as a update to the v3.2 release notes?

Reply:
Shall I list it under the existing `v3.2` section or add a `New Development` section, like this:
```
Version History
===============

New Development
---------------

* Manually pushed the dtcenter/met-base-metviewer:v3.2 image to DockerHub on 7/3/2024 to serve as the new base image for METviewer testing and development (#23)

v3.2
----

* Install SciPy Python package (#20)
* ```

Reply:
Whatever you prefer. What you have seems fine. You could instead add a bullet to v3.2 that says:

* UPDATE: 7/3/2024 Manually pushed the dtcenter/met-base-metviewer:v3.2 image to DockerHub to serve as the new base image for METviewer testing and development (#23)

I think I prefer this since the change is related to v3.2, but I am not opposed to your format.